apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/TrimDao.java

```diff
@@ -33,7 +34,10 @@ public TrimEntry save(TrimEntry trimEntry) {
                     return trimEntry;
                 }
             } else {
-                try (PreparedStatement pstm = con.prepareStatement("INSERT INTO trim (height, done) VALUES (?, ?) ")) {
+                try (PreparedStatement pstm = con.prepareStatement(
```

Comment:
Is missing 
@DatabaseSpecificDml(DmlMarker..... )
?

Reply:
same as above